api/internal/localizer/localizer_test.go

```diff
@@ -113,5 +160,161 @@ resources:
 	addFiles(t, fSysExpected, "/dst/a", map[string]string{
 		"kustomization.yaml": kustomization["Kustomization"],
 	})
-	require.Equal(t, fSysExpected, fSys)
+	checkFSys(t, fSysExpected, fSys)
+}
+
+func TestLocalizeDirInTarget(t *testing.T) {
+	for name, files := range map[string]map[string]string{
```

Comment:
I think when we get to doing `map[string]map[string]string`, it's getting a bit too complex. Suggest instead using this structure:

```
func TestLocalizeDirInTarget(t *testing.T) {
	type testCase struct {
		name  string
		files map[string]string
	}

	for _, tc := range []testCase{
		{
			name: "multi_nested_child",
			files: map[string]string{"kustomization.yaml": `apiVersion: kustomize.config.k8s.io/v1beta1
components:
- delta/epsilon
kind: Kustomization
`,
			},
		},
	} {
		t.Run(tc.name, func(t *testing.T) {
			...
		})
	}
}
```